sklearn/metrics/pairwise.py

```diff
@@ -78,6 +78,10 @@ def check_pairwise_arrays(X, Y, precomputed=False):
         True if X is to be treated as precomputed distances to the samples in
         Y.
 
+    dtype : string, type, list of types or None (default=None)
```

Comment:
OK fair enough.

For the record, when I hear "type" I am thinking "Python type" so it does sound slightly weird.